pyicloud/services/photos.py

```diff
@@ -173,6 +173,11 @@ def albums(self):
             }
 
             for folder in self._fetch_folders():
+
+                # Skiping albums having null name, that can happen sometime
+                if "albumNameEnc" not in folder['fields']:
```

Comment:
```suggestion
                if "albumNameEnc" not in folder["fields"]:
```

To fix Black formating